src/plugins/OauthPlugin/FrameStrategy/FrameStrategy.js

```diff
@@ -9,16 +9,16 @@ export default class FrameStrategy {
 
   /**
    * @param {object} params
-   * @param {boolean=} params.oauthPopup
+   * @param {boolean=} params.isPopup
```

Comment:
`boolean=`?

Reply:
it's type definition in JSDoc:
`boolean=` - means that isPopup is optional parameter

Reply:
`[param.isPopup]` – more clear